data/i18n/ar/ar.toml

```diff
@@ -0,0 +1,568 @@
+# i18n strings for the English (main) site.
+# NOTE: Please keep the entries in alphabetical order when editing
+
+# For "and", see [conjunction_1]
+
+[auto_generated_edit_notice]
+other = "(صفحة مولدة آلياً)"
+
+[auto_generated_pageinfo]
+other = """<p>هذه الصفحة مولدة آلياً،</p>
+<p>فإذا أردت الإبلاغ عن مشكلة فيها، فالرجاء ذكر ذلك، لاحتمال وجود سبب المشكلة في ملف آخر من المشروع.</p>"""
+
+[caution]
+other = "احذر:"
+
+[cleanup_heading]
+other = "جاري الترتيب"
+
+[community_events_calendar]
+other = "تقويم الفعاليت"
+
+[community_forum_name]
+other = "المنتدى"
+
+[community_github_name]
+other = "GitHub"
+
+[community_slack_name]
+other = "Slack"
+
+[community_stack_overflow_name]
+other = "Stack Overflow"
+
+[community_twitter_name]
+other = "تويتر"
+
+[community_youtube_name]
+other = "يوتيوب"
+
+# Avoid using conjunction_1.
+# Must match the context in layouts/shortcodes/release-data.html
+# Appears on https://kubernetes.io/releases/
+# For example the "and" in "Complete 1.25 Schedule and Changelog"
+[conjunction_1]
+other = "و"
+
+[cve_id]
+other = "CVE ID"
```

Comment:
```suggestion
other = "رمز سفإ"
```

Reply:
Given the ongoing discussion about this translation—I will resolve this comment until we decide what the appropriate thing to do here is.

Reply:
I don't recommend resolving things if you're unsure whether the commenter would see the feedback as addressed.

Also, I would localize “ID”.

Reply:
Translated "ID" here: https://github.com/kubernetes/website/pull/44862/commits/24e19e942643f3f2ac50bb33aab45619fddef22a